tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -39,13 +39,12 @@ const userSharesFileOrFolderWithGroup = function (file, sharee) {
   return userSharesFileOrFolderWithUserOrGroup(file, sharee, true)
 }
 
-Given('user {string} has shared file/folder {string} with user {string}', function (sharer, elementToShare, receiver) {
+const shareFileFolder = function (elementToShare, sharer, receiver, shareType = 0, permissions = 31) {
   const params = new URLSearchParams()
-  params.append('shareType', '0') // 0 = user, 1 = group, 3 = public (link), 6 = federated (cloud share).
+  params.append('shareType', shareType) // 0 = user, 1 = group, 3 = public (link), 6 = federated (cloud share).
```

Comment:
can we use strings here? the numbers are confusing and hard to remember, same for permissions

Reply:
I think It would be better to do that in new PR.